File: android/src/main/kotlin/ru/kardenvan/flutter_yandex_ads_plus/platform_api/PlatformApiConfig.kt
package ru.kardenvan.flutter_yandex_ads_plus.platform_api

/**
 * Class containing configuration values for platform communication.
 */
class PlatformApiConfig {
    companion object {
        /**
         * Ad view types
         */
        const val bannerAdViewTypeId: String = "banner_yandex_ad"
        const val nativeAdViewTypeId: String = "native_yandex_ad"

        /**
         * Method channel name
         */
        const val methodChannelName: String = "flutter_yandex_ads_plus_method_channel"

        /**
         * Event channel name
         */
        const val eventChannelName: String = "flutter_yandex_ads_plus_event_channel"
    }
}

File: lib/platform_api/platform_api_config.dart
/// Class containing configuration values for platform message communication.
///
class PlatformApiConfig {
  const PlatformApiConfig._();

  /// Ad view types

  static const String bannerAdViewTypeId = 'banner_yandex_ad';
  static const String nativeAdViewTypeId = 'native_yandex_ad';

  /// Method channel name
  static const String methodChannelName =
      'flutter_yandex_ads_plus_method_channel';

  /// Event channel name
  static const String eventChannelName =
      'flutter_yandex_ads_plus_event_channel';
}
